tools/src/test/scala/caliban/tools/ClientWriterViewSpec.scala

```diff
@@ -10,10 +10,10 @@ import zio.test.environment.TestEnvironment
 
 object ClientWriterViewSpec extends DefaultRunnableSpec {
 
-  val gen: String => RIO[Blocking, String] = (schema: String) =>
+  val gen: String ⇒ RIO[Blocking, String] = (schema: String) ⇒
```

Comment:
Can you undo these?

Reply:
Ah yes, accidentally used a different formatter - should be good now :)